src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss

```diff
@@ -1,3 +1,5 @@
+$fontMontserrat: 'Montserrat', sans-serif;
```

Comment:
do we have this font in typography folder/ file ?

Reply:
yes, thanks. i've changed it